content/blog/2021-06-30-june-21-community-gems.md

```diff
@@ -0,0 +1,142 @@
+---
+title: June '21 Community Gems
+date: 2021-06-30
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+picture: 2021-06-30/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/june-21-community-gems/779
+tags:
+  - Community
+  - Plots
+  - Pipelines
+  - CML
+  - Git
```

Comment:
`Gems` I see that this tag is present in the previous posts, should we add it here?

Reply:
@flippedcoder I know we discussed this, but I'm not sure what is the end result

Reply:
@shcheklein I haven't seen that `Gems` tag in the previous posts, so I'd leave it out. Was there a particular post you're seeing this on?

Reply:
@flippedcoder 

Hmm .. This is from the https://dvc.org/blog/may-21-community-gems:

<img width="500" alt="Screen Shot 2021-07-01 at 4 50 51 PM" src="https://user-images.githubusercontent.com/3659196/124201301-8a918000-da8c-11eb-8908-05d5ff96ccc2.png">

Reply:
I've been confused by tags too. Currently used ones:

<details><summary>tags</summary>

- Ambassador
- Azure
- Best Practices
- Cache
- CI/CD
- CML
- Community
- Data
- DevOps
- Discord
- Documentation
- DVC
- DVC Studio
- Gems
- Git
- GitHub Actions
- Google Drive
- Google Season of Docs
- Google Summer of Code
- Hacktoberfest
- Heartbeat
- Import
- Machine Learning
- Meetup
- MinIO
- MLOps
- Model Ensembling
- Open Source Summit
- Performance
- Pipelines
- Plots
- Productivity
- Project
- PyCon
- PyData
- Python
- R
- Rclone
- Release
- RStats
- SciPy
- shtab
- Students
- Tags
- Udemy Course
- Volunteer

</details>

Reply:
I see. I just made up the `Gems` tag for the last post, it wasn't there before. I wouldn't mind going back and adding `Gems` to all of the previous posts so that they get grouped together.

Maybe we should always have `Gems`, `Discord`, `DVC`, `CML` on the Community Gems posts and the others are dependent on the questions.

What do y'all think about that? It doesn't seem like there have been any particular ways tags were used on these posts before so it'd be nice to standardize it a little.

Reply:
Ah, I see. No worries. Let's merge it and may be even fix the previous post?